_includes/manuals/1.7/3.2.3_installation_scenarios.md

```diff
@@ -157,3 +157,14 @@ foreman-installer \
 {% endhighlight %}
 
 Also ensure here that the dhcp-interface argument is updated for the interface to run DHCP on.
+
+#### Manual installation of Smart Proxy
+
+To integrate Foreman Proxy with existing services, manual configuration needs
+to be done. In that case install the `foreman-proxy` package and proceed with
+editing `/etc/foreman-proxy` configuration files. The service name to restart
+is named `foreman-proxy`.
+
+We don't provide details about manual configuration, so the recommended way is
```

Comment:
isn't that the main smart proxy config chapter?  I'm not entirely sure I understand this.. :)